languages/julia/concepts/vectors/links.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
There doesn't seem to be a manual page covering _only_ 1-dim arrays/vectors. The general page on [Arrays](https://docs.julialang.org/en/v1/manual/arrays/) has a lot of info about multi-dim arrays which may seem rather overwhelming at this point in the track.